### ISSUE TITLE
Missing SLE modules

### ISSUE THREAD

**Issue by vijaigan** [stdevel/ansible-uyuni#22]
Hi 

I have tried installing the SUSE Manager 4.0 using the Ansible. 
Encountered error during the installation and found the modules needs to be updated with the following order. 
requesting you to kindly update the code base with the changes.

```
sles_modules:

  - name: sle-module-basesystem
    identifier: sle-module-basesystem/15.1/x86_64
  - name: sle-module-web-scripting
    identifier: sle-module-web-scripting/15.1/x86_64
  - name: sle-module-python2
    identifier: sle-module-python2/15.1/x86_64
  - name: sle-module-suse-manager-server
    identifier: sle-module-suse-manager-server/4.0/x86_64
  - name: sle-module-server-applications
    identifier: sle-module-server-applications/15.1/x86_64
  - name: sle-module-desktop-applications
    identifier: sle-module-desktop-applications/15.1/x86_64
  - name: sle-module-development-tools
    identifier: sle-module-development-tools/15.1/x86_64
```

**Comment by stdevel** [comment]
Hey @vijaigan,
thanks a lot for sharing - I will have a look at it. :+1: 

Best wishes,
Christian.

**Comment by stdevel** [comment]
Hey @vijaigan,
are you sure that you really need `sle-module-desktop-applications` and `sle-module-development-tools`? The [documentation](https://documentation.suse.com/external-tree/en-us/suma/4.0/suse-manager/installation/install-vm.html#jeos.susemgr.prep) isn't listing those modules.

Do you have any error message you chould share?

Best wishes,
Christian.

**Comment by stdevel** [comment]
Hi @vijaigan,
I was just reproducing this issue with SLE 15 and SUSE Manager 4.1.
There was definitely an error during the product order. But I had no issues with missing "Development Tools" and "Desktop Applications" module. I also cross-checked with official SUSE Manager 4.0 and 4.1 documentation.

Is it possible that you were installing additional things and therefore the issue was forced?

**Comment by stdevel** [comment]
Hi @vijaigan,
forget my last comment :smile: 

I just did some additional testing and we're both right. "Desktop Applications" and "Development Tools" are required for CEFS. The required package `perl-Text-Unidecode` is part of the development packages.

Thanks a lot for pointing out - just fixed this!

**Comment by stdevel** [comment]
Fixed in commit 41d0a43.
